tests/typ/visualize/gradient-math.typ

```diff
@@ -66,3 +65,27 @@ $ attach(
   Pi, t: alpha, b: beta,
   tl: 1, tr: 2+3, bl: 4+5, br: 6,
 ) $
+
+---
+
```

Comment:
General style nit: I'd prefer not blank lines between "---" and the comment describing the test.